Wikipedia/Localizations/en.lproj/Localizable.strings

```diff
@@ -587,6 +587,7 @@
 "places-search-saved-articles" = "Saved articles";
 "places-search-suggested-searches-header" = "Suggested searches";
 "places-search-recently-searched-header" = "Recently searched";
+"places-search-articles-that-match" = "Articles matching \“$1\”";
```

Comment:
Are those directional quotes (or whatever they're called) or is GitHub just making them italics?